staker/reward_math.gno

```diff
@@ -74,7 +82,7 @@ func rewardMathComputeInternalRewardAmount(tokenId uint64, deposit Deposit) bigi
 	rewardRatio := getRewardRatio(stakedDuration)
 
 	finalRewardQ := myWholeRewardX96 * bigint(rewardRatio)
-	finalReward := finalRewardQ / Q96 / Q96 / Q96 / Q96 / Q96 / 100 / 100
+	finalReward := finalRewardQ / consts.Q96 / consts.Q96 / consts.Q96 / consts.Q96 / consts.Q96 / 100 / 100
```

Comment:
I'm not familiar with the specific context or behind story of this code, but it appears that replacing `100 / 100` with `1000` might be a straightforward adjustment. 

Additionally, converting `blockRewardInit` to a `bigint` type seems feasible for use in this calculation, but I don't know if that variable can be used in a formula.

Reply:
>  but it appears that replacing 100 / 100 with 1000 might be a straightforward adjustment.

Actually, it goes same with bunch of `Q96` division. Right below that line there are some comments about why we need this kind of division formula.
https://github.com/gnoswap-labs/gnoswap/blob/64fc5ac2b64fec64e66af6bdcf96a0204b0aa105/staker/reward_math.gno#L87-L93
I kind wanted to comments to be similar as formula and this is exact reason why I didn't use `1000`. However I'm fine with your suggestion (I'll leave it you :D)

Reply:
> Additionally, converting blockRewardInit to a bigint type 
Suggesting, wait until @mconcat to finish his uint256 then we can do something about it. How about that?